documentatie/use-cases/GSB-gebruikersdoelen.md

```diff
@@ -107,33 +107,29 @@ __uitbreidingen__:
 - OSV richt de server ook gelijk als client in. Willen wij dit ook?
 - Afzenderverificatie ontbreekt nog, want nog geen beslissing over oplossing.
 
-## GSB richt de applicatie in
+
+## De coördinator maakt de stembureaus aan.
 
 __niveau:__ gebruikersdoel, zee, 🌊
 
 ### Hoofdscenario en uitbreidingen
 
-__trigger:__
-
-- [GSB installeert de applicatie.](#gsb-installeert-de-applicatie)
-- Het GSB maakt de kandidatenlijst beschikbaar.
-- De Kiesraad maakt de verkiezingsdefinitie beschikbaar.
-
 __hoofdscenario__:
 
-1. De beheerder leest de verkiezingsdefinitie in.
-2. De beheerder leest de kandidatenlijst in.
-3. De beheerder maakt de stembureaus aan.
-4. De beheerder maakt de gebruikers aan.
+1. De coördinator importeert de lijst van stembureaus.
```

Comment:
```suggestion
1. De coördinator importeert de lijst met stembureaus.
```

Reply:
Wat mij betreft gaan we er met 'importeren' eigenlijk nog iets teveel vanuit dat de lijst met stembureaus altijd uit een bestand komt. 

Soms zijn er meerdere verkiezingen tegelijkertijd (gemeenteraad + stadsdeelverkiezingen of provinciale staten + waterschappen). Wat mij betreft zou het dan logisch zijn om de stembureaus van de ene verkiezing over te nemen bij de andere. Dat past niet helemaal in deze use cases.

Reply:
Gelijktijdige verkiezingen met dezelfde stembureaus gebeurt toch alleen bij gemeenteraad + stadsdeelverkiezingen? Of je zou ook iets moeten bouwen dat je je stembureaus voor provinciale staten over de waterschapsverkiezingen kunt verdelen.

Zal het bij de open punten van deze use case opnemen, want anders lopen we wel ver vooruit op wat we aan het bouwen zijn.

Reply:
Andere verkiezingen kunnen in theorie ook samenvallen. Als een kabinet valt dicht op GR of EP, als er een referendum is, of bij gemeentelijke herindelingen. 

Maar ben het met je eens. Dit blijven uitzonderingen. Waarschijnlijk meer werk om er iets voor te bouwen, dan voor de coordinator om twee keer hetzelfde bestand met stembureaus te importeren.